File: php/Mahasiswa.php
<?php
class Mahasiswa {
    // attribute
    private $name;
    private $nim;
    private $prodi;
    private $fakultas;
    private $foto;

    // Constructor
    public function __construct($name = '', $nim = '', $prodi = '', $fakultas = '', $foto="") {
        $this->name = $name;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
        $this->foto = $foto;
    }

    // getter and setter
    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getNIM() {
        return $this->nim;
    }

    public function setNIM($nim) {
        $this->nim = $nim;
    }

    public function getProdi() {
        return $this->prodi;
    }

    public function setProdi($prodi) {
        $this->prodi = $prodi;
    }

    public function getFakultas() {
        return $this->fakultas;
    }

    public function setFakultas($fakultas) {
        $this->fakultas = $fakultas;
    }

    public function getFoto() {
        return $this->foto;
    }

    public function setFoto($foto="") {
        $this->foto = $foto;
    }

    // destructor
    function __destruct(){}
}
?>

File: c++/Mahasiswa.cpp
#include <iostream>
#include <string>

using namespace std;

class Mahasiswa {
private:
    string name;
    string NIM;
    string prodi;
    string fakultas;

public:
    Mahasiswa() {}
    
    Mahasiswa(string name, string NIM, string prodi, string fakultas) {
        this->name = name;
        this->NIM = NIM;
        this->prodi = prodi;
        this->fakultas = fakultas;
    }

    // getter dan setter
    string getName() {
        return name;
    }

    string getNIM() {
        return NIM;
    }

    string getProdi() {
        return prodi;
    }

    string getFakultas() {
        return fakultas;
    }

    void setName(string name) {
        this->name = name;
    }

    void setNIM(string NIM) {
        this->NIM = NIM;
    }

    void setProdi(string prodi) {
        this->prodi = prodi;
    }

    void setFakultas(string fakultas) {
        this->fakultas = fakultas;
    }

    // Destructor
    ~Mahasiswa() {}
};